cockroachdb/sqlalchemy/dialect.py

```diff
@@ -256,7 +256,14 @@ def get_columns(self, conn, table_name, schema=None, **kw):
 
     def get_indexes(self, conn, table_name, schema=None, **kw):
         if self._is_v192plus:
-            return super().get_indexes(conn, table_name, schema, **kw)
+            indexes = super().get_indexes(conn, table_name, schema, **kw)
+            # We need to remove the `duplicates_constraints` value from unique indexes, otherwise
```

Comment:
Could you add one initial sentence: "CockroachDB creates a UNIQUE INDEX automatically for each UNIQUE CONSTRAINT, and there is no difference between unique indexes and unique constraints."

Reply:
Sure, done!